cmd/readpool.go

```diff
@@ -45,16 +39,28 @@ var rpInfo = &cobra.Command{
 			log.Fatalf("Failed to get read pool info: %v", err)
 		}
 
+		usd, err := zcncore.ConvertTokenToUSD(info.Balance.ToToken())
+		var bt = float64(info.Balance) / 1e10
+		if err != nil {
+			log.Fatalf("Failed to convert token to usd: %v", err)
+		}
+
 		if info.Balance == 0 {
 			fmt.Println("no tokens locked")
 			return
 		}
 
 		if doJSON {
-			util.PrintJSON(info)
+			kk := map[string]interface{}{
```

Comment:
Please name the variable to something meaningful other than kk

Reply:
Oh sorry, I forgot that during the development. Changing it.